homework012/sort_user.go

```diff
@@ -2,58 +2,121 @@ package homework012
 
 import (
 	"fmt"
+	"log"
+
+	"xorm.io/xorm"
 )
 
 // 問題5
 // User構造体を実装してください。
+type User struct {
+	Id    int    `xorm:"id"`
+	Name  string `xorm:"name"`
+	Class int    `xorm:"class"`
+	Age   int    `xorm:"age"`
+}
 
 // 問題5 を実装してください。
 // 中身を適宜変更してください。引数も変更して大丈夫です。
-func insertUsers() {
+func insertUsers(engine xorm.Engine, users []User) {
+	for _, user := range users {
+		_, err := engine.Table("user").Insert(user)
+		if err != nil {
+			log.Fatal(err)
+		}
+		fmt.Printf("< insertStudent > ID:%d 名前:%s, Class:%d, 年齢:%d\n", user.Id, user.Name, user.Class, user.Age)
+	}
+	fmt.Print("------------------------------------------------------------\n")
 }
 
 // 問題5 を実装してください。
 // 中身を適宜変更してください。引数も変更して大丈夫です。
-func ascUsers() {
+func ascUsers(engine xorm.Engine) {
+	users := []User{}
+	session := engine.Table("user")
+	err := session.OrderBy("id DESC").Find(&users)
```

Comment:
```suggestion
	err := session.Asc("id").Find(&users)
```

ASCとDESCが逆になっていると思います。
また、上記のようにも書けます。